_format/1.1/index.md

```diff
@@ -585,6 +594,100 @@ that includes a [profile alias][profile aliases]:
 > Note: Additional link types, similar to `profile` links, may be specified in
 the future.
 
+#### <a href="#document-links-link-relations" id="document-links-link-relations" class="headerlink"></a> Link relation types
+
+The following link relation types **MAY** be used as the value of the `rel` link
+parameter object member:
+
+  * `add`: The link's target points to a resource where a user agent can cause
+    an addition to the link's context.
+  * `update`: The link's target points to a resource where a user agent can cause
+    an update the link's context.
+  * `remove`: The link's target points to a resource where a user agent can cause
+    a removal of all or part of the link's context.
+  * `relate`: The link's target points to a resource where a user agent can cause
+    a direct or indirect relationship to be established with the link's context.
+
+> Note: These link relation types correspond to operations described by the
+JSON:API specification and it is expected that implementations will use them to
+reference resources that will process requests according to [the requirements for
+creating, updating or deleting resources](https://jsonapi.org/format/1.1/#crud). However, this is not a strict
+limitation on their use.
```

Comment:
Once this is in, I'd like to officially register these, which explains this note and the term "user agent" vs "client".

If they're for some reason they can't be registered, we can update the spec to say that these strings resolve to the extension-relation-types:

`https://jsonapi.org/#link-relation-add`
`https://jsonapi.org/#link-relation-update`
`https://jsonapi.org/#link-relation-remove`
`https://jsonapi.org/#link-relation-relate`

and add those anchor links.